pygeos/predicates.py

```diff
@@ -186,6 +187,32 @@ def is_missing(geometry, **kwargs):
     """
     return lib.is_missing(geometry, **kwargs)
 
+
+@multithreading_enabled
+def is_prepared(geometry, **kwargs):
+    """Returns True if a Geometry is prepared
```

Comment:
```suggestion
    """Returns True if a Geometry is prepared.  Will always return False if Geometry is None.
```

Reply:
Should we actually return True instead? 
Because why would you use `is_prepared`? To know whether you would still call `prepare`? In that case, doing `is_prepared(geoms).all()` would be useful, but doesn't work with missing values